doc/api/deprecations.md

```diff
@@ -2065,6 +2065,9 @@ method.
 
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/56966
+    description: End-of-Life.
```

Comment:
You also need to upgrade the description:

https://github.com/nodejs/node/pull/56966/files#diff-b7c42d4dc97df183538f07f68a1f6ee54068ee8860cfa053ab141e64f290af87R2076

Reply:
Nice catch! Thank you for the review @marco-ippolito